workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReaderInterfaceAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    /// Marks an interface as the Reader Interface for use by visualizers.
```

Comment:
"Visualizers" is not a known term in the GDK. ;) How about just "Marks an interface as a Reader interface."?